lib/blog_fetcher.rb

```diff
@@ -0,0 +1,31 @@
+class BlogFetcher
+  BASE_URL = 'http://blog.mayday.us/api'
+  ENDPOINTS = {
+    recent:         '/read/json?num=5',
+    press_releases: '/read/json?tagged=press%20release&num=5'
+  }
+
+  KEY_BASE = "blog_feeds"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.